### DISCUSSION TITLE
Runelite for Android

### DISCUSSION THREAD

**Discussion by ChewyBacca12** [runelite/runelite#18424] [Ideas]
What the subject says.  Wouldn't it be great to develop Runelite to work on Android devices, like smart phones or tablets?  I always feel like I'm missing out a bit when I'm on my tablet vs PC.  I'm sure the developers have talked about this, but is this something coming up in the near future?

**Comment by cdfisher** [comment]
[There are no plans for a mobile version of RuneLite](https://github.com/runelite/runelite/wiki/FAQ#will-there-be-a-runelite-mobile-client).
